src/commit-split.ts

```diff
@@ -15,32 +15,67 @@
 import {Commit} from './graphql-to-commits';
 import {relative} from 'path';
 
-interface CommitSplitOptions {
+export interface CommitSplitOptions {
+  // Defaults to './'
   root?: string;
+  // Include empty git commits: each empty commit is included
+  // in the list of commits for each path.
+  includeEmpty?: boolean;
```

Comment:
Curious what workflow you use includeEmpty for, I'm guessing to force a release for all packages?

Reply:
yeah, I'd like to extend the existing functionality (using `--allow-empty` to control the version) into the aggregate*. My personal use-case would mostly be in conjunction with the future `release-<package/path>-as` footer functionality to set versions per package:

```
chore: set version 21.2.0

release-pkgA-as: 21.2.0
release-pkgB-as: 21.2.0
// the remainder just semver bump according to Conventional Commits
```

Of course you could also auto bump all packages with
```
feat: bump minor on all packages
```

Or coerce all packages to a version with
```
chore: set all to version 7.7.7

release-as: 7.7.7
```

I'll add to the comments to that effect here.


*I haven't tested but I'm pretty sure `monorepo-tags` configuration isn't currently compatible with this workflow. Probably worth making consistent at some point?